source/compilers.py

```diff
@@ -129,7 +129,8 @@ def __init__(self, machine=None, debug=0, fcompname=None, fcompexec=None, implic
             self.popt = '-g'
             self.extra_link_args += ['-g']
             self.extra_compile_args += ['-g', '-O0']
-
+        if omp:
```

Comment:
The command line flag "-fopenmp" is gnu specific. Can you move this into the *_gfortran routines? On intel, the flag is "-qopenmp".